Library/ViewModels/SetYourPasswordViewModelTests.swift

```diff
@@ -0,0 +1,56 @@
+import XCTest
+
+@testable import Library
+import ReactiveExtensions_TestHelpers
+
+final class SetYourPasswordViewModelTests: TestCase {
+  private let saveButtonIsEnabled = TestObserver<Bool, Never>()
+  private let contextLabelText = TestObserver<String, Never>()
+  private let newPasswordLabel = TestObserver<String, Never>()
+  private let confirmPasswordLabel = TestObserver<String, Never>()
+
+  func test_init() {
+    _ = self.makeSUT(with: "test@email.com")
+
+    XCTAssertNil(self.saveButtonIsEnabled.lastValue)
+  }
+
+  func test_saveButtonIsEnabledWhenFormIsValid() {
+    let sut = self.makeSUT()
```

Comment:
Yea instead of `sut` can we call this `viewModel`.
Actually, if you can create a `setUp` function that runs after each test and updates a `private var viewModel: SetYourPasswordViewModel` for each test, then we don't have to manually write `self.makeSUT()` in each case.

Reply:
sure I can move back to the pattern that is used elsewhere in the app for cohesion sake

Reply:
So my thought process here is that having a helper method like this for creating our viewmodel and setting up the inputs/outputs would achieve the following:
1. isolate any future updates to this viewmodel to one place so we don't have to update a bunch of individual tests (similar to the setUp pattern) 
2. guarantee that each test is working with a viewmodel that has a clean state (meaning its free from changes made from other tests) lowering the chances of creating flaky tests.